api/room/schema.py

```diff
@@ -124,6 +125,19 @@ def mutate(self, info, room_id, **kwargs):
         admin_roles.update_delete_rooms_create_resource(room_id)
         update_entity_fields(exact_room, **kwargs)
 
+        new_total_feedback_views = 0
+        if 'increment_room_feedback_views' in kwargs and\
```

Comment:
Could you kindly explain what this line of code is doing?

Reply:
The code checks if the argument for `increment_room_feedback_views` is in the `updateRoom`  mutation. Using the mutation for updating other fields of the room will serve an error if the argument and that line are excluded.

Reply:
I mean `and\` part of the line

Reply:
I mean and\ part of the line

Reply:
Rather than have `noqa`, I use the backslash `\` to split the line of code into three. The `and` is to ensure all the conditions are met before the block within the `if-statement` is executed.

Reply:
Let's have a better way of doing this, the implementation would make the code difficult to read for future developers.

Reply:
I would suggest that you configure your editor to autoformat by limiting 79 characters per line.

Reply:
This current implementation is to make sure the lines do not exceed the 79 limit. To break a line of code that is not a string, the `\` is required. Would it be okay to add a comment explaining the multiple conditions in one if statement to help future developers?